scripts/install_deepchem_conda.sh

```diff
@@ -18,27 +18,22 @@ then
     echo "Installing DeepChem in current env"
 else
     export envname=$1
-    conda create -y --name $envname "python>=$python_version"
+    conda create -y --name $envname python=$python_version
```

Comment:
"python>=$python_version" install python 3.8 but many package like tensorflow doesn't support python 3.8. So, conda try to downgrade python 3.6 and it is a waste of time.